### PR TITLE
Add opera-project

### PR DESCRIPTION
**Pull request by cedriclombardot**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: opera-project/admin-bundle/0.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Opera\\AdminBundle\\OperaAdminBundle": [
+            "all"
+        ]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    }
+}
```
[the remark this answers is not in the document]
Comment (cedriclombardot) [reply opera-project/admin-bundle/0.1/manifest.json]: done

File: opera-project/core-bundle/0.1/manifest.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Opera\\CoreBundle\\OperaCoreBundle": [
+            "all"
+        ]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    }
+}
```
[the remark this answers is not in the document]
Comment (cedriclombardot) [reply opera-project/core-bundle/0.1/manifest.json]: done

**Comment by cedriclombardot** [comment]
@Nyholm done

File: opera-project/admin-bundle/0.1/config/packages/opera_easy_admin.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## This is a sample easy admin configuration
+## Copy Page Section to your own easy admin configuration
+#
+imports:
+    - { resource: "@OperaAdminBundle/Resources/config/easy_admin.yaml" }
+
+# easy_admin:
```
Comment (Nyholm) [inline opera-project/admin-bundle/0.1/config/packages/opera_easy_admin.yaml]: These comments needs to be removed. They do not bring any value. 

Consider adding a link to your documentation.

File: opera-project/core-bundle/0.1/config/routes/opera_core.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Run bin/console doctrine:schema:update or make:migrate as your prefer
+# And after uncomment
+
+# opera_core_bundle:
```
Comment (Nyholm) [inline opera-project/core-bundle/0.1/config/routes/opera_core.yaml]: Can't these be uncommented before you run the commands?

**Comment by Nyholm** [comment]
Closing due inactivity. 

#eu-fossa

**Pull request closed without merging**